src/sentry/api/endpoints/project_tagkey_details.py

```diff
@@ -34,6 +35,13 @@ def delete(self, request: Request, project, key) -> Response:
             {method} {path}
 
         """
+        if ratelimiter.is_limited(
```

Comment:
Why are you using this method of limiting instead of adding a `rate_limits` custom option to the endpoint?